airflow/contrib/example_dags/example_qubole_operator.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import airflow
```

Comment:
?

Reply:
@bolkedebruin the file `example_qubole_operator` wouldn't compile if this is not present. Just to be sure, I checked other examples and this import is present in all of them.